### ISSUE TITLE
How to swap taxonomic hierarchy for specimens in only one collection?

### ISSUE THREAD

**Issue by cjconroy** [ArctosDB/arctos#6575]
MVZ mammals wants to recognize a number of taxonomic arrangements that might disagree with other collections. E.g., putting two-toed sloths into Choloepodidae, rather than Megalonychidae where they currently are. I will have to make the hierarchy, but how to swap ONLY MVZ specimens over to that arrangement? We should be allowed to disagree across arctos about higher level taxonomy.

**Comment by Jegelewicz** [comment]
Set up a new taxonomic source and put the classifications you want in there. Then prefer that source over Arctos in manage collection. I can help if you want.

**Comment by cjconroy** [comment]
It looks like I would have to do this for every single name in MVZ mammals. I currently see Arctos 1, Worms 2. If I created MVZMamm, how to do pick that for ONLY the ones where I want to do something divergent from Arctos?

**Comment by Jegelewicz** [comment]
You don't pick and choose WITHIN your collection. If you put MVZMamm as 1 in your taxonomic sources in manage colelction, then everything you have in MVZ:Mamm that has a classification in that source will use that classification. If there is no classification in that source, arctos will use the classification in the Arctos source (if you have that selected as 2 in manage collection. You don't have to change anything in your catalog records.

Make sense?

**Comment by Jegelewicz** [comment]
The trick is that first you have to create the MVZMamm taxonomy source and populate it with whatever classifications you want to use for things that diverge from the Arctos source.

**Comment by cjconroy** [comment]
This makes sense. What if there is an existing one that I like already in there? Do I need to sort of clone it and reassign as MVZMamm?

**Comment by Jegelewicz** [comment]
> What if there is an existing one that I like already in there? Do I need to sort of clone it and reassign as MVZMamm?

If it is exactly as you want it already, you don't need to do anything except pick Arctos as your second taxonomy choice in manage collection.

**Comment by cjconroy** [comment]
How do I first create the MVZMamm taxonomy source? I'm still waiting on agreement as to what we will call it.

**Comment by dustymc** [comment]
>second taxonomy choice

Or 59th - the hope has always been that we'd end up with a bunch of eg "North American shrews according to Some Expert"-type things that could get patched together (by choosing the ones you agree with ahead of the legacy trashcan for your collection) to form a dynamic and mostly complete picture.

>create

https://github.com/ArctosDB/arctos/issues/new?assignees=&labels=Function-CodeTables&projects=&template=code-table-request.md&title=Code+Table+Request+-+

>what we will call it.

Doesn't _really_ matter as long as the description is good, but "mammals according to (person, publication, whatever)" is likely to get a lot more use than "my stuff."

**Comment by Jegelewicz** [comment]
> How do I first create the MVZMamm taxonomy source? 

That needs a code table request for a new term in https://arctos.database.museum/info/ctDocumentation.cfm?table=cttaxonomy_source

**Comment by Jegelewicz** [comment]
Closing as any further progress should come through another issue as stated above.
